scheduler/server/stateful_scheduler.go

```diff
@@ -187,6 +195,13 @@ type statefulScheduler struct {
 
 	// durationKeyExtractorFn - function to extract, from taskID, the key to use for tracking task average durations
 	durationKeyExtractorFn func(string) string
+
+	SchedAlg SchedulingAlgorithm
+}
+
+func (s *statefulScheduler) String() string {
+	return fmt.Sprintf("%s, num nodes: %d, num suspended nodes: %d, maxFlakyDuration: %s, maxLostDuration: %s",
```

Comment:
would we be interested in seeing which nodes were suspended instead of just the # of them?

Reply:
at this point, I think this is only used at scheduler startup (I added it to debug an issue with the change).  I would prefer to hold off changing till we really need it.